packages/traceability/issuer-irec-api/src/pods/certification-request/irec-certification-request.entity.ts

```diff
@@ -18,5 +18,9 @@ export class IrecCertificationRequest
 
     @Column({ default: '' })
     @IsOptional()
-    irecIssueId?: string;
+    irecIssueRequestId?: string;
+
+    @Column({ default: '' })
+    @IsOptional()
+    irecCertificateId?: string; // It is called an "Item" in IREC API
```

Comment:
So, if it has the default `''` can it really be undefined? Hence the "?" ?

Reply:
yep, agree. Will remove `?`